docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -120,19 +132,19 @@ decide that we want to first collect more street videos.
 
 .. _ref-docker-with-datasource-datapool:
 
-Process new data in your S3 bucket using a datapool
----------------------------------------------------
+Process new data in your bucket using a datapool
+------------------------------------------------
 
-You probably get new raw data from time to time added to your S3 bucket.
-In our case we added 4 more street videos to the S3 bucket.
+You probably get new raw data from time to time added to your bucket.
```

Comment:
why do we have a copy of the exact same information also here and  in docs/source/docker/integration/docker_trigger_from_api.rst ?

Reply:
Hmm, I think it is because the docker_with_datasource.rst shows the local workflow with the docker run command and the docker_trigger_from_api.rst shows the Platform/API workflow but we use the same example in both cases.